toolchain-gcc.cmake

```diff
@@ -1,13 +1,6 @@
 
-if(NOT ARCH)
-    set(ARCH i386)
-endif()
-
-# Default to Debug for the build type
```

Comment:
`gcc` and `clang`: Now 'empty' instead of `Debug` :-/

Reply:
It looks like we need to eliminate all usages of `CMAKE_BUILD_TYPE` in order to avoid bugs...
https://stackoverflow.com/questions/66079007/having-conditional-statements-on-build-type-variable-a-good-design